manager/notifier.go

```diff
@@ -668,7 +671,7 @@ func (n *notifier) sendOpsGenieNotification(op notificationOp, config *pb.OpsGen
 		Timeout: timeout,
 	}
 	resp, err := client.Post(
-		*opsgenieAPIURL,
+		postUrl,
```

Comment:
`postUrl` -> `postURL` please :) (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Sorry, didn't notice this earlier.